seed/data/dev/01100_tiers.js

```diff
@@ -2,24 +2,41 @@ var tableName = 'tiers'
 var insertData = []
 
 exports.seed = function (knex, Promise) {
-  var locations = ['COMMUNITY', 'CUSTODY', 'LICENSE']
   // Deletes ALL existing entries
   return knex(tableName).del()
     .return(knex('workload').select('id'))
     .then(function (workloads) {
       workloads.forEach(function (workload) {
-        locations.forEach(function (location) {
-          insertData.push(
-            { workload_id: workload.id, tier_number: 0, overdue_terminations_total: 0, unpaid_work_total: 0, warrants_total: 0, total_cases: 0, location: location },
-            { workload_id: workload.id, tier_number: 1, overdue_terminations_total: 1, unpaid_work_total: 0, warrants_total: 0, total_cases: 1, location: location },
-            { workload_id: workload.id, tier_number: 2, overdue_terminations_total: 1, unpaid_work_total: 1, warrants_total: 0, total_cases: 2, location: location },
-            { workload_id: workload.id, tier_number: 3, overdue_terminations_total: 1, unpaid_work_total: 1, warrants_total: 1, total_cases: 3, location: location },
-            { workload_id: workload.id, tier_number: 4, overdue_terminations_total: 2, unpaid_work_total: 0, warrants_total: 0, total_cases: 2, location: location },
-            { workload_id: workload.id, tier_number: 5, overdue_terminations_total: 2, unpaid_work_total: 2, warrants_total: 0, total_cases: 4, location: location },
-            { workload_id: workload.id, tier_number: 6, overdue_terminations_total: 2, unpaid_work_total: 2, warrants_total: 2, total_cases: 6, location: location },
-            { workload_id: workload.id, tier_number: 7, overdue_terminations_total: 2, unpaid_work_total: 1, warrants_total: 0, total_cases: 3, location: location }
-          )
-        })
+        insertData.push(
+          { workload_id: workload.id, tier_number: 0, overdue_terminations_total: 0, unpaid_work_total: 0, warrants_total: 0, total_cases: 0, location: 'COMMUNITY' },
```

Comment:
We have a constant for locations in probation rules that we use in the worker. Can you add it here rather than strings?

Reply:
done